packages/openactive-broker-microservice/src/util/item-transforms.js

```diff
@@ -0,0 +1,91 @@
+const { partition, omit } = require('lodash');
+
+/**
+ * Inverts any items that have an `individualFacilityUse` property, so that the top-level `kind` is "IndividualFacilityUse"
```

Comment:
```suggestion
 * Inverts any FacilityUse items that have an `individualFacilityUse` property, so that the top-level `kind` is "IndividualFacilityUse"
```

? Just a suggestion. Maybe there's a need to be more generic, idk